docs/prover/cli/options.md

```diff
@@ -86,6 +89,28 @@ If we want to verify only `withdraw_succeeds`, we run
 If we want to verify both `withdraw_succeeds` and `withdraw_fails`, we run
 `certoraRun Bank.sol --verify Bank:Bank.spec --rule withdraw_succeeds withdraw_fails`
 
+Alternatively, to verify both `withdraw_succeeds` and `withdraw_fails`, we could
+run `certoraRun Bank.sol --verify Bank:Bank.spec --rule withdraw*`
+
+(--exclude_rule)=
+### `--exclude_rule <rule_name_pattern>`
+
+**What does it do?**
+It is the opposite flag to {ref}`--rule` - use it to specify a list of rules that
```

Comment:
let's say what happens if a rule is matched on both? what takes precedence?

Reply:
will add some text about this this